--- origsrc/jasper-2.0.12/CMakeLists.txt	2017-03-02 12:03:10.000000000 -0600
+++ src/jasper-2.0.12/CMakeLists.txt	2017-05-04 00:06:49.015879800 -0500
@@ -90,6 +90,7 @@ option(JAS_ENABLE_OPENGL "Enable the use
 option(JAS_ENABLE_STRICT "Enable pedantic error checking" false)
 option(JAS_ENABLE_AUTOMATIC_DEPENDENCIES "Enable automatic dependencies" true)
 option(JAS_LOCAL "Enable local hacks for developers (do not enable)" false)
+option(JAS_ENABLE_APPS "Enable building of the applications" true)
 option(JAS_ENABLE_DOC "Enable building of the documentation" true)
 
 ################################################################################
@@ -368,7 +369,9 @@ endif()
 ################################################################################
 
 add_subdirectory(src/libjasper)
-add_subdirectory(src/appl)
+if (JAS_ENABLE_APPS)
+	add_subdirectory(src/appl)
+endif ()
 if (JAS_ENABLE_DOC)
 	add_subdirectory(doc)
 endif ()
